### PR TITLE
Addresses #448: News feature

### PR DESCRIPTION
**Pull request by harshit98**
Addresses issue #448 

Changes: 
-  [x] Created a list of news organisations and their URLs as csv.
-  [x] Created a tab "News" in the UI.
-  [ ] Filter news content under 'News' tab.
-  [ ] Show recent results on top
-  [x] Making sure "Tools" tab is at it's respective place. 

Demo Link: <a href="https://harshit98.github.io/susper.com/">here</a>

Kindly review @nikhilrayaprolu @Marauderer97

### CODE REVIEW

**Comment by Marauderer97** [comment]
@harshit98 please check travis

File: src/app/news.service.ts [hunk 1]
```diff
@@ -0,0 +1,28 @@
+import { Injectable } from '@angular/core';
+import { Http, Jsonp } from '@angular/http';
+import 'rxjs/add/operator/map';
+import { Observable } from 'rxjs';
+
+@Injectable()
+export class NewsService {
+
+  public sportsNews: string = 'https://newsapi.org/v1/articles?source=bbc-news&sortBy=top&apiKey=b487d01b4a044b11aba9e33c77850963';
```
Comment (mariobehling) [inline src/app/news.service.ts]: The idea of our news service is not to use APIs of some news services at this point. We simply want a filter of certain search results. Please read the issue carefully again!

**Comment by mariobehling** [comment]
Also the "Tools" item in the menu has moved and it should not move.

**Comment by harshit98** [comment]
@nikhilrayaprolu @Marauderer97 Kindly review. I have updated the preview link and also left my comment on issue regarding filtering news content. :) 

@mariobehling I have solved position of tools.

**Comment by mariobehling** [comment]
@harshit98 Test link does not show the news section.

**Comment by harshit98** [comment]
There are some css conflicts which can't be solved like this implementing a new `Settings` tab which will affect the position of `News` tab. Will open a new PR when news feature will be completed.

**Pull request closed without merging**